Application/EdFi.Ods.Api/Controllers/DataManagementControllerBase.cs

```diff
@@ -322,34 +326,35 @@ private EntityTagHeaderValue GetEtag(string etagValue)
             return new EntityTagHeaderValue(Quoted(etagValue));
         }
 
-        protected string GetResourceUrl(Guid id)
+        protected string GetApplicationUrl()
         {
-            if (_applicationUrl == null)
+            if (_applicationUrl != null)
+            {
+                return _applicationUrl;
+            }
+
```

Comment:
I know this was there already, but is this caching actually useful?  We're running controllers on a per request lifecycle (as normal) so I don't think any of our scenarios will ever even hit a case where it got calculated once and gets returned here.